_health-care/_js/components/form-elements/ErrorableRadioButtons.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import _ from 'lodash';
+
+/**
+ * A radio button group with a label.
+ *
+ * Validation has the following props.
+
+ * `label` - String for the group field label.
+ * `options` - Array of options to populate group.
+ * `required` - is this field required.
+ * `value` - string. Value of the select field.
+ * `onValueChange` - a function with this prototype: (newValue)
+ */
+class ErrorableRadioButtons extends React.Component {
+  constructor() {
+    super();
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentWillMount() {
+    this.inputId = _.uniqueId('errorable-radio-buttons-');
+  }
+
+  handleChange(domEvent) {
+    this.props.onValueChange(domEvent.target.value);
```

Comment:
Agreed, let's deal with that if it turns out that we in fact need to store boolean values